README.md

```diff
@@ -85,36 +67,48 @@ tar xvf postgrest-v0.4.3.0-osx.tar.xz
 ./postgrest postgrest.conf
 ```
 
-**In a 3rd terminal**, run Pursuance's Go backend:
+**In the second terminal**, run Pursuance's Go backend:
 
 ```
 cd $(go env GOPATH)/src/github.com/PursuanceProject/pursuance
 go build
 ./pursuance
 ```
 
+**In the third terminal**, install JavaScript dependencies and start
+Pursuance's auto-reloading dev server:
+
+```
+cd $(go env GOPATH)/src/github.com/PursuanceProject/pursuance
+npm install
+npm run build
+npm run start
+```
+
 Pursuance should now be running on <http://localhost:8080>!
 
 
-### Linux Instructions (Debian/Ubuntu or similar)
+### Linux Instructions (for Debian, Ubuntu, and related)
 
-If you don't already have Postgres 9.5 or newer installed and running,
-install it:
+If you don't already have Node 7.x installed (`node --version` will tell
+you the installed version), install Node by running:
 
 ```
-sudo apt-get install postgresql postgresql-contrib
+curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+sudo apt-get install nodejs
 ```
 
-Install JS dependencies and start Pursuance's auto-reloading dev server:
+
+If you don't already have Postgres 9.5 or newer installed and running,
+install it by running:
 
 ```
-cd $(go env GOPATH)/src/github.com/PursuanceProject/pursuance
-npm install
-npm run start
+sudo apt-get install postgresql postgresql-contrib
 ```
 
-**In another terminal**, run database migrations, download
-`postgrest`, and have `postgrest` connect to Postgres:
+Next, you'll need at least three terminals. **In the first terminal**,
```

Comment:
@kurtwall Please add 2 newlines after `three terminals.` so that this matches the macOS instructions

Reply:
:heavy_check_mark: